detection_rules/main.py

```diff
@@ -111,7 +111,10 @@ def import_rules_into_repo(input_file, required_only, directory):
         base_path = contents.get('name') or contents.get('rule', {}).get('name')
         base_path = rulename_to_filename(base_path) if base_path else base_path
         rule_path = os.path.join(RULES_DIR, base_path) if base_path else None
-        additional = ['index'] if not contents.get('data_view_id') else ['data_view_id']
+
+        # handle both rule json formats loaded from kibana and toml
```

Comment:
fix: opaque issue where the contents is in a different format depending on how the rule is loaded (kibana or from toml)